### DISCUSSION TITLE
cmake without clang dependency

### DISCUSSION THREAD

**Discussion by T-3B** [termux/termux-packages#11039] [Q&A] [answered]
I cannot install cmake without Clang, and I would like to since I use GCC.
Why cmake is dependent on Clang? And how I could use cmake without Clang (but with GCC) ?

I am used to GCC and it works perfectly on my Android phone with Termux. I want to avoid taking more space by having 2 C compilers.

```
~ $ pkg show cmake
Package: cmake
Version: 3.23.2
Maintainer: @termux
Installed-Size: 38.6 MB
Depends: clang, libarchive, libc++, libcurl, libexpat, jsoncpp, libuv, make, rhash, zlib
Homepage: https://cmake.org/
Download-Size: 5189 kB
APT-Sources: https://grimler.se/termux/termux-main stable/main aarch64 Packages
Description: Family of tools designed to build, test and package software
```

**Comment by licy183** [comment]
In [ubuntu (22.04)](https://packages.ubuntu.com/jammy/cmake), `cmake` recommends ` gcc, make ` and suggests ` cmake-doc, ninja-build, cmake-format `.
In [Arch Linux](https://git.adelielinux.org/adelie/packages/-/blob/master/system/cmake/APKBUILD), `cmake` also doesn't depend on any specific compiler.
Should `cmake` in termux suggest `clang` rather than depend on it? Any ideas? @Grimler91 @buttaface @thunder-coding

**Reply by finagolfin** [reply]
I think that would be a good idea, it was added a long time ago by Fredrik, 8f2ae5e.

**Comment by licy183** [comment, marked as answer]
Now you can use `apt install cmake --no-install-recommends` to install `cmake` without `clang`.

**Reply by T-3B** [reply]
Thank you very much for your help and work !

Greatly appreciate it.
